lib/navigation/app_controller.dart

```diff
@@ -34,8 +34,12 @@ class _MyAppState extends State<AppController> {
             ],
           animationDuration: Duration(milliseconds: 200),
           animationCurve: Curves.bounceInOut
-        ), 
+        ),
       ),
+      theme: ThemeData(
+        primaryColor: Color(0xFF0F1236),
+        accentColor: Color(0xFF45cab9),
+      )
     );
```

Comment:
definitely a better way to store colors than making them global constants! thanks for this! btw, can you also remove line 3 that imports the color constants file

Reply:
i deleted this line locally? some git issue i think isn't picking up the deleted line